DirectProgramming/C++SYCL_FPGA/Tutorials/Features/loop_initiation_interval/src/CMakeLists.txt

```
@@ -30,8 +30,8 @@ endif()
 
 if(NOT DEFINED DEVICE_FLAG)
     message(FATAL_ERROR "An unrecognized or custom board was passed, but DEVICE_FLAG was not specified. \
-                         Please make sure you have set -DDEVICE_FLAG=A10, -DDEVICE_FLAG=S10, -DDEVICE_FLAG=CycloneV or \
-                         -DDEVICE_FLAG=Agilex7.")
+                         Please make sure you have set -DDEVICE_FLAG=A10, -DDEVICE_FLAG=S10, -DDEVICE_FLAG=CycloneV, \
+                         DDEVIVE_FLAG=Agilex5, or -DDEVICE_FLAG=Agilex7.")
```

```suggestion
                         Please make sure you have set -DDEVICE_FLAG=A10, -DDEVICE_FLAG=S10, -DDEVICE_FLAG=CycloneV, \
                         DDEVICE_FLAG=Agilex5, or -DDEVICE_FLAG=Agilex7.")
```
tbh we should remove this whole DEVICE_FLAG nonsense from this code sample as it's just silly

---

Updated.